CMake/kwiver-depends-OpenCV.cmake

```diff
@@ -12,9 +12,14 @@ if( KWIVER_ENABLE_OPENCV )
 
   if( OpenCV_VERSION VERSION_GREATER "2.4.6" )
     set( USE_OPENCV True )
-    if( OpenCV_VERSION VERSION_GREATER "3.0" OR
+    if( OpenCV_VERSION VERSION_GREATER "4.0" OR
+        OpenCV_VERSION VERSION_EQUAL "4.0")
+      message(STATUS "Found OPENCV 4.x")
+      set (KWIVER_HAS_OPENCV_VER_3 True)
```

Comment:
Is it true for this block that  ..  `set (KWIVER_HAS_OPENCV_VER_3 True)`

Reply:
Yes; I'm treating those as "or newer" symbols... else everywhere that looks at that (non-trivial number of instances) would have to change to `has_ver_3 || has_ver_4` (which also requires `#if[n]def` → `#if [!]defined`...).

Reply:
I find this a bit confusing since `KWIVER_HAS_OPENCV_VER_3` implies that KWIVER has OpenCV version 3 not KWIVER has OpenCV greater than version 3.   We could rename these to `KWIVER_OPENCV_MIN_VER_3` or something similar. Or if we are dropping support for OpenCV 2 then we could just get rid of `KWIVER_HAS_OPENCV_VER_3` and assume that if you've found OpenCV you have at least version 3, but I guess dropping support for OpenCV 2 is not part of this branch.